src/wasm.h

```diff
@@ -1070,6 +1068,9 @@ class MemorySize : public SpecificExpression<Expression::MemorySizeId> {
   MemorySize() { type = Type::i32; }
   MemorySize(MixedArena& allocator) : MemorySize() {}
 
+  Type ptrType = Type::i32;
+
+  void make64();
```

Comment:
In general we don't like adding methods to these classes, to keep them minimalistic. But I can't think of a better pattern, and these are rarely-used anyhow, so sgtm.

Reply:
I tried adding an argument to the constructor, but that didn't work with the `alloc` template, so did this instead to keep it simple.